src/components/IconButton.svelte

```diff
@@ -9,16 +9,14 @@
 </script>
 
 <script lang="ts">
-  export let type: IconButtonVariant = 'ghost';
+  import type { Action } from 'svelte/action';
+
+  export let variant: IconButtonVariant = 'ghost';
```

Comment:
Renamed to `variant` to prevent conflicting `props`. Melt seems to add a `type` prop to the component

Reply:
Tweaked approach so no longer conflicts.